modules/apps/headless/headless-admin-user/headless-admin-user-impl/rest-openapi.yaml

```diff
@@ -240,6 +252,17 @@ components:
                 name:
                     example: AccountGroup Name
                     type: string
+                permissions:
+                    items:
+                        type: permission
+                    type: array
+                roleBriefs:
+                    description:
+                        The list of roles associated with this account group.
+                    items:
+                        $ref: "#/components/schemas/RoleBrief"
+                    readOnly: true
+                    type: array
```

Comment:
is this needed? I think there are no roles (also because you're not filling this in the DTO)